nss-cohort-36/bangazon-api-virtual_magicians
# Last 20 products - Homepage

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.

```shell
git checkout master
git fetch --all
git checkout gc-home
python manage.py runserver

********************************************************
Part 2 if you haven't installed the new fixture files
********************************************************

python manage.py loaddata product_type
python manage.py loaddata product.json

```

**Please continue to the REACT related PR and follow the instruction there**

# Checklist:
